web/src/utils/authoriseUser.ts

```diff
@@ -1,26 +1,47 @@
 import { toast } from "react-toastify";
+
 import { OPTIONS } from "utils/wrapWithToast";
 
 type AuthoriseUserData = {
   address: `0x${string}`;
   signature: `0x${string}`;
   message: string;
 };
+const atlasUri = import.meta.env.REACT_APP_ATLAS_URI ?? "";
+
+export function authoriseUser(authData: AuthoriseUserData): Promise<string> {
+  const query = `mutation Login($message: String!, $signature: String!) {
+    login(message: $message, signature: $signature)
+  }
+  `;
+  const variables = {
+    message: authData.message,
+    signature: authData.signature,
+  };
 
-export function authoriseUser(authData: AuthoriseUserData): Promise<Response> {
-  return toast.promise<Response, Error>(
-    fetch(`/.netlify/functions/authUser`, {
+  return toast.promise<string, Error>(
+    fetch(atlasUri, {
       method: "POST",
       headers: {
         "Content-Type": "application/json",
       },
-      body: JSON.stringify(authData),
+      body: JSON.stringify({ query, variables }),
     }).then(async (response) => {
-      if (response.status !== 200) {
+      if (!response.ok) {
         const error = await response.json().catch(() => ({ message: "Error signing in" }));
         throw new Error(error.message);
       }
-      return response;
+
+      const result = await response.json();
+
+      const token = result.data.login.accessToken;
+
+      // TODO ugly fix until a more standard return type in decided upon in atlas
+      if (token) {
```

Comment:
(ignoreable) I usually find clearer to get rid of the "special" branch and then continue with the normal flow (as opposed to having if/else blocks.

```typescript
if (!token) throw new Error(result.data)
return token;
```

Reply:
agree, will follow this pattern for future cases, in this PR it's temporary. And the structure of request isn't finalized yet. This PR will remain draft until atlas is up and running